grc/tests/test_examples.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2016 Free Software Foundation, Inc.
+#
+# This file is part of GNU Radio
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+import pytest
+import os
```

Comment:
```suggestion
import os
import pytest
```

I'm sure Pylint prefers this.